kubectl-minio/cmd/cmd-operator-create.go

```diff
@@ -0,0 +1,206 @@
+ * as published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License, version 3,
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>
+ *
+ */
+
+package cmd
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"io"
+
+	"github.com/markbates/pkger"
+	"github.com/minio/kubectl-minio/cmd/helpers"
+	"github.com/minio/kubectl-minio/cmd/resources"
+	"github.com/spf13/cobra"
+
+	apiextensionv1 "k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1beta1"
+	apiextension "k8s.io/apiextensions-apiserver/pkg/client/clientset/clientset"
+
+	appsv1 "k8s.io/api/apps/v1"
+	corev1 "k8s.io/api/core/v1"
+	rbacv1 "k8s.io/api/rbac/v1"
+	kerrors "k8s.io/apimachinery/pkg/api/errors"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/runtime/serializer"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/kubernetes/scheme"
+)
+
+const (
+	operatorCreateDesc = `
+'create' command creates MinIO Operator deployment alongwith all the dependencies.`
+)
+
+type operatorCreateCmd struct {
+	out            io.Writer
+	errOut         io.Writer
+	image          string
+	ns             string
+	nsToWatch      string
+	clusterDomain  string
+	serviceAccount string
+}
+
+func newOperatorCreateCmd(out io.Writer, errOut io.Writer) *cobra.Command {
+	i := &operatorCreateCmd{out: out, errOut: errOut}
+
+	cmd := &cobra.Command{
+		Use:   "create",
+		Short: "Create MinIO Operator deployment",
+		Long:  operatorCreateDesc,
+		RunE: func(cmd *cobra.Command, args []string) error {
+			if len(args) != 0 {
+				return errors.New("this command does not accept arguments")
+			}
+			return i.run()
+		},
+	}
+
+	f := cmd.Flags()
+	f.StringVarP(&i.image, "image", "i", helpers.DefaultOperatorImage, "MinIO Operator image")
+	f.StringVarP(&i.ns, "namespace", "n", helpers.DefaultNamespace, "If present, the namespace scope for this request ")
+	f.StringVarP(&i.serviceAccount, "service-account", "", helpers.DefaultServiceAccount, "ServiceAccount for MinIO Operator")
+	f.StringVarP(&i.clusterDomain, "cluster-domain", "d", helpers.DefaultClusterDomain, "Cluster domain of the Kubernetes cluster")
+	f.StringVarP(&i.nsToWatch, "namespace-to-watch", "", "", "Namespace where MinIO Operator looks for MinIO Instances, leave empty for all namespaces")
+
+	return cmd
+}
+
+// run initializes local config and installs MinIO Operator to Kubernetes cluster.
+func (i *operatorCreateCmd) run() error {
+	sch := runtime.NewScheme()
+	_ = scheme.AddToScheme(sch)
+	_ = apiextensionv1.AddToScheme(sch)
+	decode := serializer.NewCodecFactory(sch).UniversalDeserializer().Decode
+
+	crd, err := pkger.Open("/static/crd.yaml")
```

Comment:
Why not pick it up from the current path in the repo. The Makefile target can generate it before building the plugin,